cmd/pomerium/main.go

```diff
@@ -181,3 +191,38 @@ func parseOptions(configFile string) (*config.Options, error) {
 	}
 	return o, nil
 }
+
+func handleConfigUpdate(opt *config.Options, services []config.OptionsUpdater) *config.Options {
+	newOpt, err := parseOptions(*configFile)
+	optChecksum := opt.Checksum()
+	newOptChecksum := newOpt.Checksum()
+
+	log.Debug().
+		Str("old-checksum", optChecksum).
+		Str("new-checksum", newOptChecksum).
+		Msg("cmd/pomerium: configuration file changed")
+
+	if newOptChecksum == optChecksum {
+		log.Debug().Msg("cmd/pomerium: loaded configuration has not changed")
+		return opt
+	}
+
+	if err != nil {
+		log.Error().
+			Str("error", err.Error()).
```

Comment:
The result is essentially identical but rs log has a little wrapper for `Err(err)`. https://github.com/rs/zerolog#advanced-fields

Reply:
👍